From 020478164b46cac3b69836be897967c138de18f9 Mon Sep 17 00:00:00 2001
From: Ben Finney <bignose@debian.org>
Date: Fri, 10 Feb 2017 04:38:29 +1100
Subject: [PATCH 2/2] Add suggestions to improve the ITP report.

---
 npm2deb/templates.py | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/npm2deb/templates.py b/npm2deb/templates.py
index f88861e6..475f7451 100644
--- a/npm2deb/templates.py
+++ b/npm2deb/templates.py
@@ -93,7 +93,16 @@ X-Debbugs-CC: debian-devel@lists.debian.org
 
  FIX_ME: This ITP report is not ready for submission, until you are
  confident this package description is ready for Debian.
- .""" + description_template
+ .""" + description_template + """
+FIX_ME: Explain why this package is suitable for adding to Debian. Is
+it a dependency of some other package? What benefit does it provide
+compared to other similar packages already in Debian?
+
+FIX_ME: Explain how you intend to consistently maintain this package
+in Debian. If you are not yet a Debian member, does this package need
+a sponsor? Do you have co-maintainers? Are you a member of the Debian
+JavaScript maintainers team?
+"""
 
 LICENSES = {}
 
-- 
2.11.0

