From 4dc756ef5849bf51ee41938f33ca4680d6a0829b Mon Sep 17 00:00:00 2001
From: Suhail P <psuhailp@gmail.com>
Date: Fri, 28 Oct 2016 12:20:20 +0000
Subject: [PATCH 1/2] changing CONTROL file dependency package version format

---
 npm2deb/__init__.py | 1 +
 1 file changed, 1 insertion(+)

Index: npm2deb/npm2deb/__init__.py
===================================================================
--- npm2deb.orig/npm2deb/__init__.py
+++ npm2deb/npm2deb/__init__.py
@@ -461,7 +461,10 @@ class Npm2Deb(object):
                     mapper.append_warning('error', dep, 'dependency %s '
                                           'not in debian' % (name))
                 version = dependencies[dep].replace('~', '')
+                version = dependencies[dep].replace('^', '')
                 if version:
+                    if version.find('.x') is not -1:
+                        version = version.replace('.x', '.0')
                     if version[0].isdigit():
                         version = '>= %s' % version
                     elif version == '*' or version == 'latest':
