From 6d77d1bcba65a49c7cbe44db6cb988f441b7b4a7 Mon Sep 17 00:00:00 2001
From: Shanavas M <shanavas@disroot.org>
Date: Mon, 10 Jul 2017 16:58:11 +0300
Subject: [PATCH] bump Standards-Version to 4.0.0

---
 npm2deb/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/npm2deb/__init__.py b/npm2deb/__init__.py
index 3f7c187..9586ffb 100644
--- a/npm2deb/__init__.py
+++ b/npm2deb/__init__.py
@@ -14,7 +14,7 @@
 
 VERSION = '0.2.6'
 DEBHELPER = 9
-STANDARDS_VERSION = '3.9.8'
+STANDARDS_VERSION = '4.1.1'
 
 
 class Npm2Deb(object):
